toolium/utils/download_files.py

```diff
@@ -146,7 +146,10 @@ def compare_downloaded_file(context, file_name, expected_folder, max_wait):
     :param max_wait: max time to wait
     """
     # Get downloaded file and compare with expected file
-    template_file = os.path.join(os.getcwd(), expected_folder, file_name)
+    if expected_file_name is None:
+        template_file = os.path.join(os.getcwd(), expected_folder, file_name)
+    else:
+        template_file = os.path.join(os.getcwd(), expected_folder, expected_file_name)
```

Comment:
para unificar y no repertir la línea del join, puedes hacer antes:
`expected_file_name = file_name if expected_file_name is None else expected_file_name`